projects/vdk-core/.gitlab-ci.yml

```diff
@@ -129,6 +129,26 @@ vdk-core-release-candidate-base-image:
     changes: *vdk_core_locations
 
 
+vdk-core-release-acceptance-test:
+  stage: pre_release_test
+  before_script:
+    - cd projects/vdk-core
+  script:
+    - pip install vdk-heartbeat --extra-index-url $PIP_EXTRA_INDEX_URL
```

Comment:
nit: Maybe add `pip install -U pip` on the line above, to avoid installation issues with old pip versions.

Reply:
Great, thanks!